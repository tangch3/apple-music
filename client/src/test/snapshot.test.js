import React from 'react';
import About from '../pages/about.js'
import renderer from 'react-test-renderer'


test('renders without crashing', () => {
    const tree = renderer.create(<About/>).toJSON();
    expect(tree).toMatchSnapshot();
})