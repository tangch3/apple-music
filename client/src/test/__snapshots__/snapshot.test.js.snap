// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<div>
  <h1>
    About Us
  </h1>
  <p>
    About page
  </p>
</div>
`;
